Base/usr/share/man/man1/sleep.md

```diff
@@ -0,0 +1,19 @@
+## Name
+
+sleep - delay for a specified amount of seconds
+
+## Synopsis
+
+```**sh
+$ sleep <seconds>
+```
+
+## Description
+
+`sleep` will pause execution for specified amount of seconds.
+
+## Examples
+
+```sh
+# sleep 5
```

Comment:
`#` usually implies a privileged shell is required. `sleep` does not require `root`.

```suggestion
$ sleep 5
```

Reply:
Fixed.